studio/plugins/sketch/index.js

```diff
@@ -0,0 +1,59 @@
+const fs = require('fs')
+const path = require('path')
+const { exec } = require('child_process')
+const { setup, sendRequest, sendNotification, RPCError } = require('stdio-jsonrpc')
+const { createNewSketchFile, writeSketchFile, generateId } = require('sketch-file');
+
+function modifySketchTemplate(layers, textStyles, output) {
+  const sketchDoc = createNewSketchFile(generateId(output));
+
+  sketchDoc.document.layerTextStyles.objects = textStyles;
+  sketchDoc.pages[0].layers = sketchDoc.pages[0].layers.concat(layers);
+
+  return writeSketchFile(sketchDoc, output);
+}
+
+Promise.all([
```

Comment:
What do you think about using async/await instead of Promises? Since we currently ship node 8.4, they should work automatically. It might mean making a couple wrappers for callback-based APIs, but I usually find the code is a little easier to follow in the end. I don't feel strongly about this though, if there's a reason not to.